src/__tests__/components/fields/radio-button/radio-toggle-button.spec.tsx

```diff
@@ -142,7 +142,7 @@ describe("radio toggle button", () => {
 		expect(getRadioButtonB()).toBeDisabled();
 	});
 
-	describe("update options schema", () => {
+	describe("update options through schema", () => {
```

Comment:
the schema merge is still using the overrides here

Reply:
whoops just fixed that too